.changeset/spotty-goats-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pnpm/plugin-commands-script-runners": major
```

Comment:
This change will certainly prevent `pnpm dlx` from using the old cache, forcing it to create new ones. But that's probably it. I don't think this is a breaking change.

Reply:
you are right. Maybe we will merge it to v9 as well.